api/procedure_list/urls.py

```diff
@@ -7,4 +7,6 @@
     path('procedure-cancel', views.procedure_cancel),
     path('procedure-time-execute', views.procedure_execute),
     path('params', views.params),
+    path('department-procedures', views.procedure_aggregate),
+    path('department-all', views.get_podrazdeleniya_predural_list),
```

Comment:
По именованию пути вопрос тоже, что за department-all – не ясно. И в названии функции опечатка
Тогда так
```suggestion
    path('suitable-departments', views.get_suitable_departments),
```